docs/sections/cn_workflows.md

```diff
@@ -359,6 +359,65 @@ popper run -f sample.yml -c config.yml
 
 This runs the step `one` locally in the host and step `two` through slurm on 2 nodes.
 
+### Kubernetes
+
+Popper can be run on a cluster of multiple containers using Kubernetes as the fundamental resource manager to orchestrate the execution of a step to several nodes.
+You can get started with running Popper workflows through K8s(another way to represent Kubernetes) by following the example below.
+
+Let's consider a workflow `sample.yml` like the one shown below.
+```yaml
+version: '1'
+steps:
+- id: one
+  uses: docker://alpine:3.9
+  args: echo hello-world
+
+- id: two
+  uses: popperized/bin/sh@master
+  args: ls -l
+```
+
+To run all the steps of the workflow through kubernetes,
+use the `--resource-manager` or `-r` option of the `popper run` subcommand to specify the resource manager.
+
+```bash
+popper run -f sample.yml -r kubernetes
+```
+
+When this command is run, what happens underneath is the following: `stepname kubernetes run namespace podname`
+
+Things to note:
+
+* stepname : is the step id of the workflow
+* kubernetes run: command to run kubernetes
+* namespace: Namespaces are a way to divide cluster resources between multiple users (via resource quota).
+* podname: Group of containers
+
+You can also have more control over the steps in the kubernetes resource manager.
+To do that we create a config file `config.yml` with the specifications as shown below.
+
+```yaml
+engine:
+  name: docker
+  options:
```

Comment:
these options for docker are not really needed, since the workflow doesn't run in docker . So we can remove the `options` part.